user/customizing-the-build.md

```diff
@@ -114,7 +114,15 @@ Currently, neither the `after_success` nor `after_failure` have any influence on
 
 ## Deploying your Code
 
-An optional phase in the build lifecycle is deployment. This step can't be overridden, but is defined by using one of our continuous deployment providers to deploy code to Heroku, Engine Yard, or a different supported platform.
+An optional phase in the build lifecycle is deployment. This step can't be
+overridden, but is defined by using one of our continuous deployment providers
+to deploy code to Heroku, Engine Yard, or a different supported platform.
+
+When deploying files to a provider, prevent Travis CI from deleting all files
```

Comment:
Can't we assume that if people are deploying they are using build artefacts in most cases? 

I can talk about changes instead of files, and if you think mentioned `git stash` is important that also.